src/stores/RoomViewStore.tsx

```diff
@@ -162,6 +164,12 @@ export class RoomViewStore extends Store<ActionPayload> {
         this.__emitChange();
     }
 
+    public setFavouriteStatus = (val: boolean) => {
+        dis.dispatch({ action: Action.ViewFavouriteMessages, val });
+    };
+
+    public getFavouriteStatus = () => this.state.isFavouriteClicked;
```

Comment:
Does this need to exist?

Reply:
Ahh, i guess it's no more needed, i'll remove it, i used it previously to detect if the button is clicked in the RoomList.tsx  so i can render the FavouriteMessagesView component.